homeassistant/components/zha/core/channels/base.py

```diff
@@ -361,7 +361,7 @@ def log(self, level, msg, *args):
         _LOGGER.log(level, msg, *args)
 
 
-class EventRelayChannel(ZigbeeChannel):
+class ClientChannel(ZigbeeChannel):
     """Event relay that can be attached to zigbee clusters."""
 
     CHANNEL_NAME = CHANNEL_EVENT_RELAY
```

Comment:
Rename this too?

Reply:
Oopsie.

Reply:
I'm removing `CHANNEL_NAME` here, but in next PR gonna get rid of `CHANNEL_NAME` completely, as we no longer use it.